packages/mint-css/typography/fonts/font-face.css

```diff
@@ -0,0 +1,23 @@
+@font-face {
+  font-family: 'GrowwSans';
+  font-style: normal;
+  font-weight: 300 700;
+  src: url(./GrowwSans-Variable.woff2) format('woff2-variations');
+}
+
+@font-face {
```

Comment:
@prashant-adhikari-groww lets send NotoSans in the first version itself ?

Reply:
HNS will be affected due to this. Currently app haven't made it live

Reply:
@shubhamgavali1998 Lets pull back NotoSans from this version, we'll send it in later releases/version

Reply:
Removed

Reply:
Removed